recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "MSDF font atlas generator"
+    topics = ("conan", "msdf-atlas-gen", "msdf", "font", "atlas")
+    settings = "os", "compiler", "arch"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("artery-font-format/1.0")
+        self.requires("msdfgen/1.9.1")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self._source_subfolder)
+
+    def _patch_sources(self):
+        cmakelists = os.path.join(
+            self._source_subfolder, "CMakeLists.txt")
+        if not os.path.exists(cmakelists):
+            tools.tools_files.rename("CMakeLists_internal.txt", cmakelists)
+
+        tools.replace_in_file(cmakelists,
+                              "add_subdirectory(msdfgen)", "")
+        tools.replace_in_file(cmakelists,
+                              "target_link_libraries(msdf-atlas-gen PUBLIC Threads::Threads msdfgen::msdfgen msdfgen::msdfgen-ext)",
+                              "target_link_libraries(msdf-atlas-gen PUBLIC Threads::Threads ${CONAN_LIBS})")
+        tools.save_append(cmakelists,
+                          "install(TARGETS msdf-atlas-gen-standalone DESTINATION bin)")
+
+    def _get_or_configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["MSDF_ATLAS_GEN_BUILD_STANDALONE"] = True
+        self._cmake.definitions["MSDFGEN_USE_OPENMP"] = False
+        self._cmake.definitions["MSDFGEN_USE_CPP11"] = True
+        # No need to install msdfgen or build its standalone, we just want the msdf-atlas-gen executable
+        self._cmake.definitions["MSDFGEN_INSTALL"] = False
+        self._cmake.definitions["MSDFGEN_BUILD_MSDFGEN_STANDALONE"] = False
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._get_or_configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        cmake = self._get_or_configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "msdf-atlas-gen"
+        self.cpp_info.names["cmake_find_package_multi"] = "msdf-atlas-gen"
+
+    def deploy(self):
+        self.copy("*", dst="bin", src="bin")
```

Comment:
No `deploy` please + don't set the `cmake_find_package` generator names since the upstream project does not create one either.
```suggestion
        cmake.install()

    def package_info(self):
        bin_path = os.path.join(self.package_folder, "bin")
        self.output.info("Appending PATH environment variable: {}".format(bin_path))
        self.env_info.PATH.append(bin_path)
```